pyre/check.py

```
@@ -2393,14 +2398,15 @@ def _warm_wasm_cache(self):
         print("Warming wasmtime module cache (.cwasm)...")
-        try:
-            subprocess.run(
-                [runner, "--engine", "wasmtime", os.devnull],
-                stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL,
-                env=env, timeout=120,
-            )
-        except (OSError, subprocess.SubprocessError):
-            pass  # best-effort; a cold first bench just pays the compile once
+        for extra in ({}, {"PYRE_WASM_JIT_STATS": "1"}):
+            try:
+                subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)